doc/source/whatsnew/v0.25.0.rst

```diff
@@ -181,6 +181,15 @@ The repr now looks like this:
     }]
     json_normalize(data, max_level=1)
 
+Column order is preserved when passing a list of dicts to DataFrame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
We should maybe put it in the API changes section? As some people do rely on column order

Reply:
done.